src/components/VueTablePage.vue

```diff
@@ -162,12 +162,12 @@ export default {
   },
   data () {
     return {
-      show: this.hideInfoBtn ? true : this.showMessage,
+      show: this.message.length ? this.showMessage : false,
```

Comment:
Clever, although I would never use a number as a condition like this and would still do the explicit `===  0` check. Don't know if your implementation is considered bad practice or not (or if it has performance benefits).

Reply:
It's not uncommon to use 0, undefined and so on as falsey. I think the laxness is due to the javascript language. I think now it just a preference of coding style.